### DISCUSSION TITLE
How to Get Started?

### DISCUSSION THREAD

**Discussion by najak3d** [CommunityToolkit/Maui.Markup#16] [Q&A] [answered]
Do you have a "Getting Started" blurb?   Should we be using VStudio 2019 or 2022 Preview?   Maui-Check 0.10.0?  Any of that?

Just wondering ahead, the best way to get started, so we don't spend hours spinning our wheels trying to "do it wrong."

A few paragraphs stating the tools/workloads/etc required for this to work would be great (for both just "using" the nuget, as well as for those wanting to compile from source).   We're very interested in getting away from XAML, in favor of C# markup (for both MVVM and MVU).

**Comment by TheCodeTraveler** [comment, marked as answer]
Hi @najak3d! 

We are currently working on the docs and will have a Getting Started section for `CommunityToolkit.Maui.Markup` when we move this toolkit from pre-release to stable alongside .NET MAUI in the next few months. The docs are open-source and you can follow our progress here: https://github.com/MicrosoftDocs/CommunityToolkit

In the mean-time, I've written a blog post that should help:
https://codetraveler.io/2021/10/18/using-net-maui-community-toolkit-2/

I've also included a sample app in this repo that you can reference: https://github.com/CommunityToolkit/Maui.Markup/tree/main/samples

**Reply by TheCodeTraveler** [reply]
Hi @najak3d! The official docs are now live: [https://docs.microsoft.com/en-us/dotnet/communitytoolkit/maui/markup/markup](https://docs.microsoft.com/en-us/dotnet/communitytoolkit/maui/markup/markup?WT.mc_id=mobile-0000-bramin).

We are continuing to write the docs and they will be complete before our scheduled release date in May.

**Comment by CliffAgius** [comment]
Closed and answered
